FilteringApp.sln

```diff
@@ -11,9 +11,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FilteringApp.Core", "Filter
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "GCD", "GCD", "{A1FD278A-5635-44D0-9EFC-286EA44DA7F6}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GrandCommonDivisor.ConsoleUI", "GrandCommonDivisor\GrandCommonDivisor.ConsoleUI.csproj", "{897C70BC-A580-40C9-A8D6-563706AF89EB}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GrandCommonDivisor.ConsoleUI", "GCD\GrandCommonDivisor\GrandCommonDivisor.ConsoleUI.csproj", "{897C70BC-A580-40C9-A8D6-563706AF89EB}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GrandCommonDivisor.Core", "GrandCommonDivisor.Core\GrandCommonDivisor.Core.csproj", "{EBF254E7-33E6-429C-AB40-836AAEF26232}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GrandCommonDivisor.Core", "GCD\GrandCommonDivisor.Core\GrandCommonDivisor.Core.csproj", "{EBF254E7-33E6-429C-AB40-836AAEF26232}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "UnitTests", "UnitTests", "{EB546BC7-12CC-4C80-8088-E475B3BA1043}"
```

Comment:
Usually we add unit test project next to the project that you test, so that your structure should be like
- FilteringApp folder
--FilteringApp.ConsoleUI project
--FilteringApp.Core project
--FilteringApp.Tests project

Same for GCD

Reply:
done.